pkg/binary/envoy/controlplane/istio.go

```diff
@@ -39,6 +40,10 @@ const (
 	initialEpochBootstrap = "envoy-rev1.json"
 )
 
+// envoyBootstrapJson is the "envoy_bootstrap_v2.json" from the Istio release tag or distribution
+//go:embed istio-1.2.5/tools/packaging/common/envoy_bootstrap_v2.json
```

Comment:
If we feel we need to validate this externally, we now can. For now, I think it is unlikely to drift and so we don't need to check it on each change. If a problem later happens, we can use @basvanbeek suggestion to validate it (ex on make check), but I think it would be distracting to add that now since the path says it all.